terratorch/datamodules/generic_pixel_wise_data_module.py

```diff
@@ -23,6 +23,20 @@ def wrap_in_compose_is_list(transform_list):
     # set check shapes to false because of the multitemporal case
     return A.Compose(transform_list, is_check_shapes=False) if isinstance(transform_list, Iterable) else transform_list
 
+def load_from_file_or_attribute(value:list[float] | str):
+
+    if type(value) == list:
+        return value
+    elif type(str): # It can be the path for a file
+        if os.path.isfile(value):
+            try:
+                content = np.genfromtxt(value).tolist()
+            except:
+                raise Exception(f"File must be txt, but received {value}")
+        else:
+            raise Exception("It seems that {value} does not exist or is not a file.")
```

Comment:
same issue as for the scalar_label. Probably makes sense to make this a common function both use.